pkg/webui/console/components/events/index.js

```diff
@@ -30,6 +30,7 @@ import Icon, {
   IconClipboardCheck,
 } from '@ttn-lw/components/icon'
 import Button from '@ttn-lw/components/button'
+import Routes from '@ttn-lw/components/switch'
```

Comment:
```suggestion
import Switch from '@ttn-lw/components/switch'
```
That confused me good 😂